README.md

```diff
@@ -49,34 +49,6 @@ Thus 'master' will support the upcoming major version and the current stable.
 The latest release (git tag, Puppet Forge) should support current and the
 previous stable release.
 
-### Foreman 1.11 compatibility notes
```

Comment:
I think it would be helpful for users if we tell them which version they should use on 1.11.